source/whats-new.txt

```diff
@@ -35,6 +36,26 @@ Learn what's new in:
 * :ref:`Version 3.7 <version-3.7>`
 * :ref:`Version 3.6 <version-3.6>`
 
+.. _version-5.4:
+
+What's New in 5.4
+-----------------
+
+New features of the 5.4 {+driver-short+} release include:
+
+.. important:: Deprecation Notice
+
+  - The ``collStats`` helper is deprecated. Use the :manual:`$collStats
+    </reference/operator/aggregation/collStats>` aggregation operator instead.
+  - Methods of ``db.command()`` that were incorrectly included in the
+    typescript interface have been deprecated.
+
+- The ``ChangeStream.tryNext`` method now uses the ``TChange`` schema
+  generic instead of the untyped ``Document`` interface.
```

Comment:
S: Something like:

```suggestion
- The ``ChangeStream.tryNext`` method now uses the schema-specific
``TChange`` generic type instead of the ``Document`` interface.
```